include/pmacc/identifier/value_identifier.hpp

```diff
@@ -49,19 +49,23 @@
  * @{
  */
 
-/** @param ... must be a device function/lambda with arguments worker and  IdGenerator
+/** @param ... Must be a device function/lambda with arguments worker, IdGenerator and source particle.
+ *             @attention source particle can be invalid, it is required to check the state via particles
+ *             isHandleValid() method.
+ *             To stay generic the lambda should be guarded against deriving from a particle which does not provide the
+ *             required attributes to initialize the value.
```

Comment:
Just for my understanding, do these lines mean the same as 
To stay generic the lambda does not check if the source particle provides the required attributes to initialize the value.

Reply:
No what I like to say is that the author of the lambda for the value_identiier is responsible to check and handle cases where the source particle is not providing the required attributes to derive the data.